packages/fuels/src/cli/utils/getLatestFuelsVersion.ts

```diff
@@ -0,0 +1,32 @@
+import { checkAndLoadCache, FUELS_VERSION_CACHE_TTL, saveToCache } from './fuelsVersionCache';
+
+export const getLatestFuelsVersion = async (): Promise<string | undefined> => {
+  const now = Date.now();
+
+  const cache = checkAndLoadCache();
+  if (cache !== null && cache.data !== null && now - cache.timestamp < FUELS_VERSION_CACHE_TTL) {
```

Comment:
```suggestion
  if (cache !== null) {
```

I believe we do these checks within `checkAndLoadCache`